doc/integrator/object_editing.rst

```diff
@@ -45,3 +45,25 @@ Features
         * Cut from: Cut the feature with a geometry from another layer. The
           layer needs WFS and a copyable Metadata.
     * FeatureQuery showing a FeatureWindow
+
+Configuration
+-------------
+
+The layer must be editable.
+
+Copy / cut: The layer from where one wants to copy / cut from has to be:
+
+    * Available as WFS layer
+    * Metadata "copyable" (checked) set on the layer
+
+Add the following configuration int the ``vars.yaml`` file to change the triangle size:
+
+.. code:: yaml
+
+   vars:
+     interfaces_config:
+       oeedit:
+         constants:
+           <<: *constants
+           gmfObjectEditingToolsOptions:
+             regularPolygonRadius: 150
```

Comment:
For my understanding: above we say that we are changing the triangle's size, is it really a triangle or any polygon?

Reply:
As I know it's only the triangle size, the name is like it because to draw a triangle we draw a regular polygon with 3 edges, and we can draw only one type of regular polygon...